openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -58,16 +58,12 @@ def process(self, instance):
         else:
             members = instance[:]
 
-        loaded_containers = None
-        if set(self.add_for_families).intersection(
-                set(instance.data.get("families")),
-                set(instance.data.get("family").lower())):
-            loaded_containers = self._add_loaded_containers(members)
-
         selection = members
-        if loaded_containers:
-            self.log.info(loaded_containers)
-            selection += loaded_containers
+        if set(self.add_for_families).intersection(
+                set(instance.data.get("families"))) or \
```

Comment:
Be aware that `families` _might_ not exist and you can't get make a set of `None`.